fs/remote/resolver_test.go

```diff
@@ -335,3 +335,33 @@ func (r *retryRoundTripper) RoundTrip(req *http.Request) (res *http.Response, er
 	}
 	return
 }
+
+func TestCustomUserAgent(t *testing.T) {
+	// Note: The version is set at link time, so the version will be
+	// "<unknown>" when running unit tests unless
```

Comment:
The version is set at link time, so the version will be "<unknown>" when running unit tests unless we include the vcs information in `go test` like we do for `go build`.